src/main/java/org/example/subject.java

```diff
@@ -1,2 +1,22 @@
-package org.example;public class subject {
+package org.example;
+
+public class subject {
```

Comment:
Javaのクラス名は大文字始まりです。
```suggestion
public class Subject {
```
Javaの命名規則について学習しましょう！

Reply:
クラス命名規則はパスカルケースであり、最初を大文字で書き始めます。